GitTfs/Core/ExportWorkItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Sep.Git.Tfs.Core
+{
+    public class ExportWorkItem : IExportWorkItem
+    {
+        public ExportWorkItem(ITfsWorkitem wi = null)
+        {
+            if(wi!=null)
+            {
+                this.Id = wi.Id.ToString();
+                this.Title = wi.Title;
+            }
+        }
```

Comment:
When I read this the first time, I wondered why it would make sense to build an `ExportWorkItem` with no work item from TFS. As I scrolled down, I saw that you don't actually want to create an `ExportWorkItem` without id+title. Can you change this so there are two constructor choices for `ExportWorkItem`?

```csharp
public ExportWorkItem(ITfsWorkitem wi)
{
  this.Id = wi.Id.ToString();
  this.Title = wi.Title;
}

public ExportWorkItem(string id, string title)
{
  this.Id = id;
  this.Title = title;
}
```